### PR TITLE
TW 517 Cycle 4 edits

### PR DESCRIPTION
**Pull request by HeatherStenson**
This PR covers the following style guide changes:

- “You will see” (don’t use)
- Element (instead of entity or component)
- UI (don’t use)
- Time references like now, soon, and new (don’t use)
- etc (don’t use)
- eg (use for example)
- easy, simple, just (don’t use)
- Let’s and we (avoid in LC docs)
- Via (don’t use)
- Trash (not trash can, trash icon, or trash button)
- kill (avoid)

### CODE REVIEW

File: src/pages/docs/sending-requests/authorization.md [hunk 1]
```diff
@@ -343,7 +343,7 @@ The official AWS Signature documentation provides more detail:
     * If you select __Request Headers__, Postman will add `Authorization` and `X-Amz-` prefixed fields in the __Headers__ tab.
     * If you select __Request URL__, Postman will add the auth details in __Params__ with keys prefixed `X-Amz-`.
 
-1. Enter your access key and secret values either directly in the fields or via variables for additional security.
+1. Enter your access key and secret values directly in the fields. For additional security, you can use [variables](/docs/sending-requests/variables/) for these values.
```
Comment (claireriley) [inline src/pages/docs/sending-requests/authorization.md:346]: works either way but as an option
```suggestion
1. Enter your access key and secret values directly in the fields. For additional security, you can use [secret variables](/docs/sending-requests/variables/#variable-types) for these values.
```

**Review by claireriley — APPROVED** [review]
Looks great! Most of those "via"s were me, apologies! 😅

File: src/pages/docs/collaborating-in-postman/using-workspaces/conflicts.md [hunk 2]
```diff
@@ -28,19 +28,19 @@ contextual_links:
...
+You can choose to preserve your local changes, or update your local version to match the synced version from the server. There is an overview of each conflict. Choose __Local changes__ or __Server changes__ for each one.
 
-After you choose which version to use, select __Resolve Conflicts__ to update both your local version and the synced version of the projects you're working on. You will need to make selections for every collection affected and select __Resolve Conflicts__ to update each one. If you choose your local changes, other devices and team members will now see your changes whenever Postman syncs for them.
+After you choose which version to use, select __Resolve Conflicts__ to update both your local version and the synced version of the projects you're working on. You will need to make selections for every collection affected and select __Resolve Conflicts__ to update each one. If you choose your local changes, other devices and team members will see your changes whenever Postman syncs for them.
```
Comment (mark-anthony-postman) [inline src/pages/docs/collaborating-in-postman/using-workspaces/conflicts.md:43]: Do we need to replace "see" here? Maybe "will get your changes" or "will receive your changes" or "can view your changes"? Or maybe this is a case where "see" is appropriate

File: src/pages/docs/getting-started/proxy.md [hunk 3]
```diff
@@ -117,7 +117,7 @@ If you are unable to send any requests through Postman and your network requires
...
     * **Windows** - Select **Start**, then select the gear icon (**Settings**) at the far left. In Settings, select **Network & Internet**. In the left-hand pane, select **Proxy** at the bottom. Select **Manual Proxy Setup** > use a Proxy **(ON)**.
 
-    * **Mac** - Open System Preferences and select **Network**. On the left-hand side, make sure to select the connected or active network connection. You can have different proxy settings for each type of network connection. At the bottom, select **Advanced**. Select the **Proxies** tab and you will see different protocols you can configure. If you select **Web Proxy (HTTP)**, you will be able to enter the proxy server IP address, port number, username, and password.
+    * **Mac** - Open System Preferences and select **Network**. On the left-hand side, make sure to select the connected or active network connection. You can have different proxy settings for each type of network connection. At the bottom, select **Advanced**. Select the **Proxies** tab to see different protocols you can configure. If you select **Web Proxy (HTTP)**, you will be able to enter the proxy server IP address, port number, username, and password.
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/proxy.md:120]:
```suggestion
    * **Mac** - Open System Preferences and select **Network**. On the left-hand side, make sure to select the connected or active network connection. You can have different proxy settings for each type of network connection. At the bottom, select **Advanced**. Select the **Proxies** tab to configure different protocols. If you select **Web Proxy (HTTP)**, you will be able to enter the proxy server IP address, port number, username, and password.
```

**Review by mark-anthony-postman — APPROVED** [review]
Wow, another epic round of awesome changes! 🤺 Thanks so much for continuing to spiff up the Learning Center! 💎✨

File: src/pages/docs/getting-started/proxy.md (hunk 3 above)
Comment (HeatherStenson) [reply src/pages/docs/getting-started/proxy.md:120]: Thanks for catching all these @mark-anthony-postman! I searched specifically for "you will see" and didn't catch a lot of other instances of "see" 🤦

File: src/pages/auto-flex-policy.jsx [hunk 4]
```diff
@@ -82,12 +82,12 @@ class AutoFlexPolicyPage extends React.Component {
...
-                quarter of your annual Subscription Term. You add 4 new Users two months into your
-                billing cycle. You deprovision 2 Users before the next regular billing date (i.e.
+                quarter of your annual Subscription Term. You add four new Users two months into your
+                billing cycle. You deprovision two Users before the next regular billing date (in other words,
                 the end of the third month of your annual Subscription Term). You will be charged
-                for the 2 incremental Users you added during the preceding three month period and
+                for the two incremental Users you added during the preceding three month period and
                 retained as of the billing date, at the prorated price for the remaining nine months
-                of your annual Subscription Term (i.e. $216 total).
+                of your annual Subscription Term (in this example, $216 total).
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/auto-flex-policy.jsx:90]: Good call! @Kalpana-Postman are these changes to https://learning.postman.com/auto-flex-policy/ okay?

File: src/pages/docs/administration/managing-your-team/configuring-domain-capture.md [hunk 5]
```diff
@@ -82,15 +82,15 @@ In the domain dashboard, you can view your domains, their verification status, t
...
+> You won't be able to selectively pick users you'd like to add to your team. When domain capture is enabled, any user associated with the domain, who can authenticate with the SSO auth method you've enabled, can join your Postman team.
 
 With domain capture enabled, any new users that sign up for Postman with your domain are automatically added to your team and can be managed in your [team dashboard](https://go.postman.co/settings/team/members).
 
 ## User experience
 
 When domain capture is enabled, Postman users with accounts associated with the domain or subdomain will only be able to access Postman after joining your Postman team. This applies to existing users who are a part of other Postman teams, existing individual Postman users, and new users who sign up for Postman.
 
-Existing users will be notified when they next sign in to Postman that their account is now managed by your team.
+The next time existing users sign in to Postman, they will get a notification that their account is managed by your team.
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/administration/managing-your-team/configuring-domain-capture.md:93]:
```suggestion
The next time existing users sign in to Postman, they will get a notification that their account is now managed by your team.
```

File: src/pages/docs/running-collections/working-with-data-files.md [hunk 6]
```diff
@@ -30,19 +30,21 @@ You can use data files to pass Postman sets of values to use in a collection run
...
 ## Running collections with data files
 
-When you [initiate a collection run](/docs/running-collections/intro-to-collection-runs/) you will see an option to select a data file. Select __Runner__ at bottom right of Postman.
+You can select a data file to use in a [collection run](/docs/running-collections/intro-to-collection-runs/).
 
 ![Collection Runner](https://assets.postman.com/postman-docs/collection-runner-for-v8.gif)
 
-Select your collection, then select __Select File__ next to __Data__ on the right to select your data file.
+1. Select <img alt="Runner icon" src="https://assets.postman.com/postman-docs/icon-runner-v9.jpg#icon" width="16px"> __Runner__ from the Postman footer.
+1. Select your collection and drag it into the **Run Order** workspace.
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/running-collections/working-with-data-files.md:38]:
```suggestion
1. Select your collection and drag it into the **Run Order** work area.
```

File: src/pages/docs/collaborating-in-postman/using-workspaces/conflicts.md (hunk 2 above)
Comment (HeatherStenson) [reply src/pages/docs/collaborating-in-postman/using-workspaces/conflicts.md]: Thanks for catching this! Turns out I missed a lot of "sees" on my first pass 🙈

File: src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md [hunk 7]
```diff
@@ -46,13 +46,13 @@ You can also [view who's active in your workspace](/docs/collaborating-in-postma
 
 ### Accessing the activity feed from Postman
 
-To access the activity feed in Postman, select the Overview tab and go to the **Activity** section to see the events that have occurred within the workspace.
+To access the activity feed in Postman, select the Overview tab and go to the **Activity** section learn about events that have occurred within the workspace.
```
Comment (HeatherStenson) [inline src/pages/docs/collaborating-in-postman/using-workspaces/changelog-and-restoring-collections.md:49]:
```suggestion
To access the activity feed in Postman, select the **Overview** tab. Go to the **Activity** section to learn about events that have occurred within the workspace.
```

File: src/pages/docs/collaborating-in-postman/using-workspaces/conflicts.md [hunk 8]
```diff
@@ -28,19 +28,19 @@ contextual_links:
 warning: false
 ---
 
-By syncing your Postman data with an account, you can work between devices and as part of a team. If a conflict occurs between the data in your local app and the data synced to the Postman servers, you will see a prompt allowing you to choose which version to preserve.
+By syncing your Postman data with an account, you can work between devices and as part of a team. If a conflict occurs between the data in your local app and the data synced to the Postman servers, you will be prompted to choose which version to preserve.
```
Comment (HeatherStenson) [inline src/pages/docs/collaborating-in-postman/using-workspaces/conflicts.md:31]:
```suggestion
By syncing your Postman data with an account, you can work between devices and as part of a team. If a conflict occurs between the data in your local app and the data synced to the Postman servers, Postman will prompt you to choose which version to preserve.
```

File: src/pages/docs/publishing-your-api/custom-doc-domains.md [hunk 9]
```diff
@@ -35,7 +35,7 @@ In the Postman web dashboard, select [Team Settings](https://go.postman.co/setti
 
 [![edit view for team](https://assets.postman.com/postman-docs/edit-team-profile-a.jpg)](https://assets.postman.com/postman-docs/edit-team-profile-a.jpg)
 
-In the __Custom Domains__ section, you will see a list of domains indicated with verification status. Select __Add custom domain__.
+In the __Custom Domains__ section, there is a list of domains indicated with verification status. Select __Add custom domain__.
```
Comment (HeatherStenson) [inline src/pages/docs/publishing-your-api/custom-doc-domains.md:38]:
```suggestion
The __Custom Domains__ section contains a list of domains with the verification status of each. Select __Add custom domain__.
```

File: src/pages/auto-flex-policy.jsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/auto-flex-policy.jsx]: I'm reverting this for now since changes to this page should be part of a bigger convo.

**Pull request merged**